pyproject.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.12,<3.13"
 
+django-staff-sso-client = "^4.3.0"
```

Comment:
I'm not a big fan of using internal packages for something that we have django-socials for, but we're already unique in the login methods for this project that it won't make too much difference

Reply:
@nboyse is this DBT specific, or okta? who else could use this (in principle)?